src/draw.cpp

```diff
@@ -843,7 +852,8 @@ int KDraw::is_forestsquare(int fx, int fy)
         return 0;
     }
     auto mapseg = map_seg[Game.Map.Clamp(fx, fy)];
-    switch (mapseg) //FIXME: The indexes of overworld forest tiles should come from either a map (.tmx) or a script (.lua).
+    switch (
```

Comment:
`clang-format` appears to mangle this line as well. We may need to move the comment line so it's happy.